tests/test_manifest.py

```diff
@@ -4,7 +4,10 @@
 import mock
 import os
 import pytest
-from StringIO import StringIO
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
``six`` has something for this, but there's not enough 2/3 weirdness to _really_ need to bring it in